integ/scripts/bash/run-e2e-automated.sh

```diff
@@ -5,6 +5,8 @@
 
 # Used as the entry point to run the tests in an automated way. This expects the following env vars to be set:
 # USER_ACCEPTS_SSPL_FOR_RFDK_TESTS - Needs to be 'true' to use these tests
+#
+# Each of these env vars must be set to assume an IAM role. If at least one is not set, an IAM role will not be assumed.
```

Comment:
Maybe clarify that this is to assume a role in a separate account. If you would like to assume a role in the same account, we'd need to do some extra work to make the external id optional. I don't see that use case being required for anything though.

Reply:
Sounds good. Added this clarification. and yeah, I don't think we'll need to make the external ID optional any time soon.